_includes/sections/browser-recommendation.html

```diff
@@ -66,7 +66,7 @@ <h1 id="browser-android" class="anchor">
 {% include cardv2.html
 title="Bromite"
 image="/assets/img/svg/3rd-party/bromite.svg"
-description='Bromite is a Chromium-based browser with security enhancement patches from GrapheneOS and other security-focused projects, built-in adblocking, and DNS over HTTPS support. More info can be found <a href="https://www.bromite.org/#main-features">on their website</a>.'
+description='Bromite is a Chromium-based browser with privacy and security enhancements, built-in adblocking, DNS over HTTPS support; it includes patches from ungoogled-chromium and other privacy-focused projects. More info can be found <a href="https://www.bromite.org/#main-features">on the official website</a>.'
```

Comment:
```suggestion
description='Bromite is a Chromium-based browser with privacy and security enhancements, built-in adblocking, DNS over HTTPS support; it includes patches from ungoogled-chromium and other privacy-focused projects. More info can be found <a href="https://www.bromite.org/#main-features">on the official website</a>.'
```

I think there is an "and" missing.

Reply:
Where would you add it? Before `DNS over HTTPS`?

Reply:
Yeah I’d think so

Reply:
Is there a reason we don’t say patches also come from GrapheneOS?

Reply:
@nitrohorse let's reverse the question: why would you want to mention GrapheneOS in the first place? Is it a non-technical eason?

GrapheneOS is the youngest of the mentioned projects and the one from which I picked the least patches compared to the others; it includes some Bromite patches itself and sometimes create others based on Bromite's patches (see [this comment](https://github.com/GrapheneOS/Vanadium/pull/48#issuecomment-567731962) for a recent example), so there is no clear downstream/upstream (more of a collaboration), while for ungoogled-chromium there is a lot of patches in use in Bromite (~11). I keep correct patch author attribution as much as I can and that helps to identify contributions; there are 3 important patches taken from GrapheneOS (64-bit WebView processes, -fstack-protector-strong, -fwrapv in clang), thus I know it's not one I would mention first to be fair vs other projects which contributed more patches.

P.S. it should be Vanadium anyways, since GrapheneOS is not the browser

Reply:
Apologies for the late reply, I missed this. I guess I was under the impression Bromite started from the GrapheneOS project which is why I was thinking it’s still helpful to list GOS in the description. But thanks for clarifying, this change makes sense 👍🏼

Reply:
No problem, I thought it was odd. Bromite is ~2 years older than the GrapheneOS project and Daniel (the author behind GrapheneOS) also refers to Bromite here: https://grapheneos.org/usage

This is because of the ongoing free-form collaboration.